cmd/cycles.go

```diff
@@ -31,10 +31,11 @@ var cyclesCmd = &cobra.Command{
 	Short: "Prints cycles in dependency chains.",
 	Long:  `Will show all the cycles in the dependencies of the project.`,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		overview := getDepInfo()
```

Comment:
I think the old signature was easier to reason about than modifying inputs...

`overview := getDepInfo(nil)` can default the main modules to the first LHS found.

`overview := getDepInfo([]string{"foo","bar","baz"}) can use "foo","bar","baz" as the main modules